pkg/gen/typegen.go

```diff
@@ -252,6 +252,14 @@ func makeTypescriptType(prop map[string]interface{}, opts groupOpts) string {
 		ref == "io.k8s.apimachinery.pkg.apis.meta.v1.MicroTime" {
 		// TODO: Automatically deserialized with `DateConstructor`.
 		return stringT
+	} else if ref == "io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.JSONSchemaPropsOrBool" {
+		return "apiextensions.v1beta1.JSONSchemaProps | boolean"
```

Comment:
Actually we should remove this function altogether, because it's dead code. I'd originally meant for @swgillespie to pick up this function and use it to make mypy types, but I think we should just come back and re-add this when we're ready.

Reply:
I pushed another commit that removes it.